cluster/gce/config-default.sh

```diff
@@ -305,9 +305,6 @@ if [[ "${NODE_OS_DISTRIBUTION}" == "gci" ]]; then
 else
   export ENABLE_NODE_PROBLEM_DETECTOR="${KUBE_ENABLE_NODE_PROBLEM_DETECTOR:-daemonset}"
 fi
-NODE_PROBLEM_DETECTOR_VERSION="${NODE_PROBLEM_DETECTOR_VERSION:-}"
```

Comment:
isn't still being used in NPD? https://github.com/kubernetes/node-problem-detector/blob/f42281ee2658900bdb0571e1159a43f6ab712a19/test/build.sh#L84 Can you please explain why it is not needed any longer?

Reply:
Thanks for the comment. I wasn't aware of this before. 
From my understanding, NPD used NODE_PROBLEM_DETECTOR_VERSION for new version's staging testing, therefore in k8bs's downloading logic,  NODE_PROBLEM_DETECTOR_VERSION can override DEFAULT_NPD_VERSION when it is set.  In this case, we need this logic for testing purpose, and doesn't need clean up.

Reply:
CC: @vteratipally

So this PR should be closed?

Reply:
Thanks @SergeyKanzhelev for noticing the NPD e2e test dependency on NODE_PROBLEM_DETECTOR_VERSION. In that case, I think we should keep NODE_PROBLEM_DETECTOR_VERSION. We should still upgrade NPD in k/k from v0.8.5 https://github.com/kubernetes/kubernetes/blob/03709c0/cluster/gce/gci/configure.sh#L29 to latest version v0.8.7